---
title: Research
---
I am interested in understanding how multicellular life is programmed into genomic DNA. Every cell in an organism contains the same genome with the same genes, but different types of cells express different sets of genes. This phenomenon is orchestrated by *cis*-regulatory sequences (CRS), non-coding DNA sequences that specify when, where, and how much genes are expressed. Most of the variation between individuals and between species can be attributed to genetic variation within CRS. In fact, most disease-associated variants lie not in genes, but between the genes, and exert their influence through effects on gene expression.

Despite their importance in development, disease, and evolution, how CRS function is poorly understood. What combinations of DNA sequence features specify a CRS? Does the same CRS behave differently in other cellular environments, such as by increasing gene expression in one cell type and decreasing expression in another cell type? Can we predict *cis*-regulatory activity from DNA sequence? Can we manipulate DNA sequences to quantitatively tune the activity of a CRS? And because there are so many possible variables to manipulate, how can we design the most informative experiments?

To answer these questions, I use Massively Parallel Reporter Assays to test thousands of CRS at once. Then, I use a combination of techniques from systems biology, data science, machine learning, and active learning to search for patterns in the data and design future rounds of experiments.
